src/series/ops/image.rs

```diff
@@ -1,19 +1,52 @@
-use crate::datatypes::{DataType, ImageType};
+use arrow2;
+use image;
+
+use crate::array::ops::image::DaftImageBuffer;
+use crate::datatypes::{logical::ImageArray, DataType, ImageType, NullArray};
 
-use crate::series::IntoSeries;
 use crate::{
     error::{DaftError, DaftResult},
-    series::Series,
+    series::{IntoSeries, Series},
 };
 
 impl Series {
     pub fn image_decode(&self) -> DaftResult<Series> {
         match self.data_type() {
             DataType::Binary => {
```

Comment:
What do you think about implement this `image_decode` method on the `BinaryArray` and have the Series dispatch to it like we do in the other series methods?